CMakeLists.txt

```diff
@@ -31,7 +32,14 @@ option(
   VELOX_BUILD_MINIMAL
   "Build a minimal set of components only. This will override other build options."
   OFF)
-
+# No string options
```

Comment:
I am not sure I understand what this comment is trying to tell. Can you expand it?

Reply:
Sure, `option()`s are always bool `ON|OFF` so we have to use a normal string type cache variable for the "option". Will expand it.